src/intelli/private/graphexecmodel_impl.h

```diff
@@ -1445,9 +1445,9 @@ struct GraphExecutionModel::Impl
                 continue;
             }
 
-            bool updatedIter = false;
+            bool updatedQueue = false;
```

Comment:
Think about renaming `updatedQueue` to currentNodeDeleted

Reply:
Done